ProfitSite/src/main/java/bsu/fpmi/profit/LoginService.java

```diff
@@ -0,0 +1,60 @@
+package bsu.fpmi.profit;
+
+import java.sql.*;
+
+public class LoginService {
+    private final String URL;
+    private final String USER;
+    private final String PASSWORD;
+    private Connection connection;
+
+    public LoginService(String url, String user, String password) {
+        this.URL = url;
+        this.USER = user;
+        this.PASSWORD = password;
+    }
+
+    public boolean connect() {
+        try {
+            Class.forName("com.mysql.cj.jdbc.Driver");
```

Comment:
The driver must only be loaded once in the whole lifecycle of the program. the connections should be created ... not once, but still in the one particular place, in the one generic method. That's what the CRUD class is for. Also, do you remember the connection pool technique?)